devicetypes/technisat/technisat-on-off-switch.src/technisat-on-off-switch.groovy

```diff
@@ -0,0 +1,527 @@
+/**
+ *	Copyright 2021 TechniSat
+ *
+ *	Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *	in compliance with the License. You may obtain a copy of the License at:
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *	Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *	on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *	for the specific language governing permissions and limitations under the License.
+ *
+ */
+import groovy.json.JsonOutput
+metadata 
+{
+	definition (name: "TechniSat On/Off switch", namespace: "TechniSat", author: "TechniSat", vid:"generic-switch-power-energy",
+				mnmn: "SmartThings", runLocally: true, minHubCoreVersion: '000.017.0012',
+				executeCommandsLocally: false)
+	{
+		capability "Energy Meter"
+		capability "Switch"
+		capability "Power Meter"
+		capability "Refresh"
+		capability "Configuration"
+		capability "Health Check"
+
+		command "reset"
+
+		fingerprint mfr: "0299", prod: "0002", model: "1A90", deviceJoinName: "TechniSat Switch"
+	}
+
+	preferences 
+	{
+		parameterMap.each 
+		{
+			input(title: "Parameter ${it.paramZwaveNum}: ${it.title}",
+			description: it.descr,
+			type: "paragraph",
+			element: "paragraph")
+			if(it.enableSwitch)
+			{
+				input(name: it.enableKey,
+				title: "Enable",
+				type: "bool",
+				required: false)
+			}
+			input(name: it.key,
+			title: it.paramName,
+			type: it.type,
+			options: it.values,
+			range: it.range,
+			required: false)
+		}
+	}
+}
+def installed() 
```

Comment:
Could you please refactor formatting so it would look like:
def installed() {
...
}
Same applies to ifs, etc.